signature/mechanism_none.go

```diff
@@ -0,0 +1,19 @@
+//go:build containers_image_disable_signing
+// +build containers_image_disable_signing
```

Comment:
Something like
```suggestion
// +build containers_image_disable_simple_signing
```
(updated everywhere) because this does not prevent use of sigstore. (Cc: @rhatdan )

Reply:
@mtrmac I suppose what I meant by `disable_signing` is "the signing mechanism is disabled so you won't be able to create new signatures" - am I correct that disabling `mechanism*.go` does this?

Reply:
c/image/signature does not _only_ do cryptography; it includes policies like “always allow” and “always reject”.

Of the cryptographic mechanisms, it now actually implements _two: “simple signing”, which requires GPG, and “sigstore”, which doesn’t. All of the discussion so far was about disabling / choosing a mechanism underlying the “simple signing” part; “sigstore” is unaffected.

So, the tag name should refer (in some way) to simple signing in particular, and not to the concept of signing in general.